downloads/swan-lake-release-notes/swan-lake-preview4/RELEASE_NOTE.md

```diff
@@ -314,6 +315,18 @@ Executing a `fail` Statement will cause the control flow to transfer to the `on
     }
 ```
 
+##### Predeclared module prefixes
+
+The module prefix `t` (where `t` could be `boolean`, `decimal`, `error`, `float`, `future`, `int`, `map`, `object`, `stream`, `string`, `table`, `typedesc`, or `xml`) has been predeclared as referring to the `lang.t` lang library module now.
```

Comment:
```suggestion
The module prefix `t` (in which `t` could be `boolean`, `decimal`, `error`, `float`, `future`, `int`, `map`, `object`, `stream`, `string`, `table`, `typedesc`, or `xml`) has been predeclared as referring to the `lang.t` lang library module now.
```

Reply:
Since `t` is the entire entity, what do refer to by "in which"?

Reply:
The complete occurrence should be referred to by "in which".

Reply:
Fixed.